### ISSUE TITLE
A:focus reset violates WCAG 2.0

### ISSUE THREAD

**Issue by CharlesBelov** [woothemes/FlexSlider#1431]
Please back out issue Should reset for a:focus #38 and find some other solution. 

.flex-container a:hover,
.flex-slider a:hover,
.flex-container a:focus,
.flex-slider a:focus {
  outline: none;
}

breaks accessibility.

2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. (Level AA)

**Comment by jeffikus** [comment]
Hi there

Have you tested with the latest version -> 2.5.0?

**Comment by CharlesBelov** [comment]
I haven't installed it, but it contains the same problematic CSS code:

2.4.7 flexslider.css lines 14-17
```
.flex-container a:active,
.flexslider a:active,
.flex-container a:focus,
.flexslider a:focus  {outline: none;}
```

2.5.0 flexslider.css contains in lines 25-30:

```
.flex-container a:hover,
.flex-slider a:hover,
.flex-container a:focus,
.flex-slider a:focus {
  outline: none;
}
```

**Comment by jeffikus** [comment]
I'll take a look for the 2.6.0 milestone, but can you post a live url here so I can test your version of the issue?

**Comment by CharlesBelov** [comment]
Currently https://www.sfmta.com/ has a prominent slider on the home page. If you tab to any of the links in the slider, there is no outline. Currently it is happening in Firefox, IE9 Windows, and Chrome.

I am about to implement a new version of the style sheet which corrects this problem by overriding the styles via our own style sheet, so please let me know when you've seen this so I can proceed.
